lib/screens/profile_screen.dart

```diff
@@ -1,12 +1,33 @@
 import 'package:flutter/material.dart';
+import 'package:jamaica/widgets/collected_item_screen.dart';
 
 class ProfileScreen extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Scaffold(
-        appBar: AppBar(
-          title: Text('Profile'),
+    return MaterialApp(
```

Comment:
Dont create a MaterialApp inside any class. It should be created only once in the main